src/cms/database/migrations/2024.03.01T00.00.10.footer-menu.js

```diff
@@ -0,0 +1,142 @@
+    "publishedAt": "2024-02-28T20:00:52.023Z",
+    "isExternalUrl": null,
+    "parent_id": null,
+  },
+  {
+    "id": 3,
+    "title": "Stay connected",
+    "url": null,
+    "order": 3,
+    "createdAt": "2024-02-28T20:01:08.932Z",
+    "updatedAt": "2024-02-28T20:01:14.896Z",
+    "publishedAt": "2024-02-28T20:01:14.890Z",
+    "isExternalUrl": null,
+    "parent_id": null,
+  },
+  {
+    "id": 4,
+    "title": "Park-use permits",
+    "url": "/park-use-permits/",
+    "order": 1,
+    "createdAt": "2024-02-28T20:02:21.754Z",
+    "updatedAt": "2024-02-29T00:44:03.372Z",
+    "publishedAt": "2024-02-28T20:02:23.565Z",
+    "isExternalUrl": false,
+    "parent_id": 1,
+  },
+  {
+    "id": 5,
+    "title": "Filming in parks",
+    "url": "/park-use-permits/filming-in-parks/",
+    "order": 2,
+    "createdAt": "2024-02-28T20:03:11.438Z",
+    "updatedAt": "2024-02-29T00:44:12.753Z",
+    "publishedAt": "2024-02-28T20:05:00.216Z",
+    "isExternalUrl": false,
+    "parent_id": 1,
+  },
+  {
+    "id": 6,
+    "title": "Travel trade",
+    "url": "/park-use-permits/travel-trade/",
+    "order": 3,
+    "createdAt": "2024-02-28T20:05:49.900Z",
+    "updatedAt": "2024-02-29T00:44:18.653Z",
+    "publishedAt": "2024-02-28T20:05:50.901Z",
+    "isExternalUrl": false,
+    "parent_id": 1,
+  },
+  {
+    "id": 7,
+    "title": "Donate",
+    "url": "/get-involved/donate/",
+    "order": 1,
+    "createdAt": "2024-02-28T20:06:36.277Z",
+    "updatedAt": "2024-02-29T00:44:25.134Z",
+    "publishedAt": "2024-02-28T20:06:37.432Z",
+    "isExternalUrl": false,
+    "parent_id": 2,
+  },
+  {
+    "id": 8,
+    "title": "Buy a licence plate",
+    "url": "/get-involved/buy-licence-plate/",
+    "order": 2,
+    "createdAt": "2024-02-28T20:08:19.223Z",
+    "updatedAt": "2024-02-29T00:44:31.469Z",
+    "publishedAt": "2024-02-28T20:08:20.055Z",
+    "isExternalUrl": false,
+    "parent_id": 2,
+  },
+  {
+    "id": 9,
+    "title": "Volunteer",
+    "url": "/get-involved/volunteer/",
+    "order": 3,
+    "createdAt": "2024-02-28T20:08:54.946Z",
+    "updatedAt": "2024-02-29T00:44:38.028Z",
+    "publishedAt": "2024-02-28T20:08:56.012Z",
+    "isExternalUrl": false,
+    "parent_id": 2,
+  },
+  {
+    "id": 10,
+    "title": "Contact us",
+    "url": "/contact/",
+    "order": 1,
+    "createdAt": "2024-02-28T20:11:12.168Z",
+    "updatedAt": "2024-02-29T00:44:43.803Z",
+    "publishedAt": "2024-02-28T20:11:18.011Z",
+    "isExternalUrl": false,
+    "parent_id": 3,
+  },
+  {
+    "id": 11,
+    "title": "BC Parks blog",
+    "url": "https://engage.gov.bc.ca/bcparksblog/",
+    "order": 2,
+    "createdAt": "2024-02-28T20:11:48.149Z",
+    "updatedAt": "2024-02-28T23:52:54.317Z",
+    "publishedAt": "2024-02-28T20:11:49.430Z",
+    "isExternalUrl": true,
+    "parent_id": 3,
+  }
+]
+
+async function up(knex) {
+
+  let i, j, rowsToInsert, chunk = 200;
+
+  // insert rows in 200 row batches
+  if (await knex.schema.hasTable('footer_menus')) {
```

Comment:
We'll also need to create a "post deployment tasks" ticket for this release with a step to delete the migration from Postgres and restart one of the CMS pods.  

The table won't get created until AFTER the migration is run, so hasTable will return "false" the first time it runs

Reply:
https://bcparksdigital.atlassian.net/browse/CM-1323